pkg/controller/argocd/argocd_controller.go

```diff
@@ -66,36 +63,16 @@ func newReconciler(mgr manager.Manager) reconcile.Reconciler {
 func add(mgr manager.Manager, r reconcile.Reconciler) error {
 
 	reqLogger := logs.WithValues()
-	reqLogger.Info("Watching ArgoCD")
+	reqLogger.Info("Watching ArgoCD Server Route")
 
-	// Skip controller creation if ArgoCD CRD is not present
-	_, err := mgr.GetRESTMapper().RESTMapping(schema.GroupKind{
-		Group: argocdGroup,
```

Comment:
You don't need either of these constants now?

`argocdKind` or `argocdGroup`

Reply:
Removed the unused constants